tensorflow/c/c_api.cc

```diff
@@ -1344,7 +1344,8 @@ void TF_OperationGetAttrString(TF_Operation* oper, const char* attr_name,
         InvalidArgument("Attribute '", attr_name, "' is not a string");
     return;
   }
-  if (max_length <= 0) {
+  if (max_length == 0) {
+    status->status = InvalidArgument("Attribute '", max_length, "' is zero");
```

Comment:
We had an internal review of this and unfortunately this breaks the API a bit since we never enforce in the documentation the max_length needs to be a certain size. Could you instead just remove this if condition altogether and we'll just do a memcpy of 0 bytes?

Reply:
Sure, thanks for pointing out. I actually removed this in my last commit. Not sure why its still there. I will revert back this change. Thanks again.